proto_v2/txn_v2.proto

```diff
@@ -0,0 +1,66 @@
+syntax = "proto3";
+package fd.v2;
+
+import "instr_v2.proto";
+
+/* Message header information. */
+message TxnHeader {
+    uint32 num_required_signatures = 1;
+    uint32 num_readonly_signed_accounts = 2;
+    uint32 num_readonly_unsigned_accounts = 3;
+}
+  
+/* Address Lookup table related metadata. This is used to parse the data from 
+   the lookup table accounts. from the account list. */
+message LUTEntry {
+    bytes           account_key      = 1;
+    repeated uint32 writable_indexes = 2;
+    repeated uint32 readonly_indexes = 3;
+}
+
+/* As a note, if the scope of the fuzzer is just an instruction, then all of
+   the fields in TxnEnv will be set to null except for account_keys and 
+   InstrEnv. TODO: maybe aluts too for instructions. This should all be
+   prepopulated probably */
```

Comment:
ALUTs should not be a part of instruction fuzzing. Resolution happens at txn level, and when we execute the instruction the addresses should already be loaded in as standard accounts within the harness

Reply:
good point